javascript/javascript2/week3/game.js

```diff
@@ -0,0 +1,92 @@
+const gameTime = document.getElementById("secondsForGame");
+const btn = document.getElementById("startButton");
+const gameOver = document.getElementById("gameOver");
+const win = document.getElementById("winner");
+const playerOnLead = document.getElementById("playerOnLead")
+
+let clickCountS;
+let clickCountL;
+let keySelect;
+
+btn.addEventListener('click', startGame)
+
+function startGame() {
+    window.addEventListener('keydown', playerfunc)
+    clickCountS = document.getElementById("clickCountS");
+    clickCountL = document.getElementById("clickCountL");
+    keySelect = document.getElementById("keySelection");
+
+    let keyS = 0;
+    let keyL = 0;
+    function playerfunc(e) {
+
+        if (e.keyCode == '83') {
+            clickCountS.innerText = keyS++;
+
+        } else if (e.keyCode == '76') {
+            clickCountL.innerText = keyL++;
+        }
+        else {
+            keySelect.innerText = `Select between 'S' or 'L' keys`
+            // alert(`Select between 'S' or 'L' keys`)
+        }
+
+        //extra feature
+        if (keyS > keyL) {
+            playerOnLead.innerHTML = `Player S on lead`
+        }
+        if (keyS < keyL) {
+            playerOnLead.innerHTML = `Player L on lead`
+        }
+        //
+        if (winnerObj.sPlayer) {
+            clickCountS.innerText = winnerObj.sPlayer;
+        }
+        if (winnerObj.lPlayer) {
+            clickCountL.innerText = winnerObj.lPlayer;
+        }
+        if (winnerObj.sPlayer > winnerObj.lPlayer) {
+            win.innerHTML = "Congrats Player S";
+
+        }
+        if (winnerObj.sPlayer < winnerObj.lPlayer) {
+            win.innerHTML = "Congrats Player L";
+        }
+        if (winnerObj.sPlayer == winnerObj.lPlayer) {
+            win.innerHTML = "It's a draw, Try Again";
+        }
```

Comment:
Use some blank lines to separate these if statements.
```suggestion
        if (keyS > keyL) {
            playerOnLead.innerHTML = `Player S on lead`
        }
        if (keyS < keyL) {
            playerOnLead.innerHTML = `Player L on lead`
        }
        //
        if (winnerObj.sPlayer) {
            clickCountS.innerText = winnerObj.sPlayer;
        }
        else if (winnerObj.lPlayer) {
            clickCountL.innerText = winnerObj.lPlayer;
        }

        if (winnerObj.sPlayer > winnerObj.lPlayer) {
            win.innerHTML = "Congrats Player S";
        }
        else if (winnerObj.sPlayer < winnerObj.lPlayer) {
            win.innerHTML = "Congrats Player L";
        }
       else {
            win.innerHTML = "It's a draw, Try Again";
        }
```

Reply:
Thanks a lot Cameron..will change it right away